src/PubSub.ts

```diff
@@ -14,6 +14,8 @@ export class PubSub implements PubSubEngine {
   private handlers: Map<string, Handler> = new Map();
   private nextSubscriptionId: number = 0;
   private subscriptions: Map<number, Unsubscribe> = new Map();
+  private argsForHandlers: Map<string, any> = new Map();
+  private subscriptionIdsWithTopic: Map<number, any> = new Map();
```

Comment:
To be honest, I would find it much nicer if we just had a property here that contained all the information (args, unsubscribe, topic). Otherwise you always have to be careful with the implementation that you don't forget some element.

Reply:
Great suggestion. 

I appreciate the feedback. I'll fix it up later this morning and send it back your way.  

Thanks for taking the time to review!